### DISCUSSION TITLE
How to make a *.msi file before the v0.5.0 version?

### DISCUSSION THREAD

**Discussion by sun363587351** [microsoft/ebpf-for-windows#2481] [Q&A] [answered]
Are there any steps to compile and generate msi files by yourself before version v0.5.0？

**Comment by dthaler** [comment, marked as answer]
No, we don't expect (or desire) anyone to use older pre-release versions as there have been many bugs fixed since then.

**Reply by sun363587351** [reply]
ok,thanks
